app/src/main/java/com/example/myapplication/ui/LoginFragment.kt

```diff
@@ -81,14 +82,24 @@ class LoginFragment : Fragment() {
     private fun validEmailObserver() {
         loginViewModel.validEmail.observe(viewLifecycleOwner) {
             if (it == true) {
-                this.findNavController().navigate(
-                    LoginFragmentDirections.actionLoginFragmentToHomePageFragment()
-                )
+                // this.findNavController().navigate(
+                //     LoginFragmentDirections.actionLoginFragmentToHomePageFragment()
+                // )
                 loginViewModel.resetValidEmailValue()
```

Comment:
the same comment about reseting values applies here, every time you are reseting an observable, you must question yourself that is for a good reason. If you can't find a good reason to reset, it's because you simply don't need to :)